agent/bench-scripts/pbench-cyclictest

```diff
@@ -123,6 +123,7 @@ printf "\n" >>$benchmark_summary_html_file
 export benchmark config
 
 count=1
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir beg
```

Comment:
We don't need this one, as the `beg` parameter makes it a noop.  This is true for all invocations of `pbench-metadata-log`.

We should remove the `beg` and `end` parameters from `pbench-metadata-log`, modifying that script to stop checking them.